### ISSUE TITLE
Version 3.0 release

### ISSUE THREAD

**Issue by maximilianmeier** [firebase/emberfire#602]
Since the version 3.0 is already available as a release candidate, i want to ask what is still open or need to be done until the version can be released?

If there are ToDos left I would try to help with some of them, because i just can't wait for the update to be released.

Feel free to add a ToDo-List and then maybe other people can help as well.

ToDos:
- [ ] Realtime Updates not working
- [ ] Ember Simple Auth not working
- [ ] Fastboot not working

**Comment by maxymczech** [comment]
A good place to start would be to finish this: https://github.com/firebase/emberfire/blob/master/docs/migration/2XX-to-3XX.md

**Comment by maximilianmeier** [comment]
I'm curious if @jamesdaniels has already changes in a private branch. Because I want to avoid adding documentation for example that is already done by him. But If there is nothing prepared that is probably a good start thanks @maxymczech  👍

**Comment by marendra** [comment]
ember simple auth and fastboot still dont work in emberfire, please someone fix this, thanks
